### PR TITLE
chore: Add light menu background color

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Layout inline sub menu add background color to help distinguish level. Add `@menu-inline-submenu-bg` less variable and rename `@menu-dark-submenu-bg` to `@menu-dark-inline-submenu-bg`.      |
| 🇨🇳 Chinese |      Layout 内联子表单添加背景颜色以更好的区分层级。新增 `@menu-inline-submenu-bg` 变量，且 `@menu-dark-submenu-bg` 改名为 `@menu-dark-inline-submenu-bg`。      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/menu/style/index.less [hunk 1]
```diff
@@ -542,6 +542,7 @@
 
   &-sub&-inline {
     padding: 0;
+    background: @menu-submenu-bg;
```
Comment (afc163) [inline components/menu/style/index.less:545]:
```suggestion
    background: @menu-inline-submenu-bg;
```

**Comment by zombieJ** [comment]
设计师那边颜色不做的太深免得太喧宾夺主。和 dark 模式一样都有了背景色。

File: components/menu/style/index.less (hunk 1 above)
Comment (zombieJ) [reply components/menu/style/index.less:545]: 这个对应是 `@menu-dark-submenu-bg`

Comment (zombieJ) [reply components/menu/style/index.less:545]: 要不都统一成 `menu-dark-inline-submenu-bg` 吧？

Comment (afc163) [reply components/menu/style/index.less:545]: dark？这个不是 light 的么

Comment (zombieJ) [reply components/menu/style/index.less:545]: 就是原本暗色下有一个变量叫做 `menu-dark-submenu-bg`，于是亮色照着这个名字起了一个 `menu-submenu-bg`。你这边推荐 `menu-inline-submenu-bg`，我想是不是就干脆一起都改掉加上 `-inline` 更语义化一些。

Comment (afc163) [reply components/menu/style/index.less:545]: 可以的。

**Review by xiaohuoni — APPROVED** [review]

File: components/style/themes/dark.less [hunk 2]
```diff
@@ -290,7 +290,7 @@
 // Menu
 // ---
 // dark theme
-@menu-dark-submenu-bg: @component-background;
```
Comment (yoyo837) [inline components/style/themes/dark.less:293]: 不需要保留兼容吗？

Comment (yoyo837) [reply components/style/themes/dark.less:293]: 自定义样式覆盖的

Comment (zombieJ) [reply components/style/themes/dark.less:293]: changelog 里写一下，看看反馈，多了再 patch，免得用的人少还多了一个历史债。

**Pull request merged**